### PR TITLE
Chore : Code Coverage, Code Simplify

### PR DESCRIPTION
**Pull request by yezz123**
# Code Coverage, Code Simplify 🚀

- Hello @Abdenasser 👋, I'm happy to contribute to your project, by Opening This Pull Request.
- I see that We can Minify the code and make it more readable, by reducing Code Specification and Use the pep8 code requirements for The Coverage.

## Overview

- I will try to show the Function that I edited in the code and the code that I added to the code.

### Function `Generator.get_fields_string`

- Simplify generator expression.

> An identity generator `(a for a in coll)` can be replaced directly with the collection coll.

### Function `is_present_in_file`

- Simplify conditional into return statement

#### Explanation

- The expression `(a for a in b)` is a generator that returns all of the elements of `b`. If being passed into a function like any or all that takes a generator or sequence, it can simply be replaced by `b` which is much clearer.

### Test Cases

- in the `tests` folder, i find in the most of Test Cases Functions this little Style issue `Simplify generator expression` , and i try to simplify them.

- Function `TestGenerator.test_run` , Instead, create a new class and place the fields and methods responsible for the relevant functionality in it.

- Function `TestGenerator.test_generate_models` , here you can Extract duplicate code into method.

> One of the main ways to remove duplication is to extract the common areas into another method and call that.

### CODE REVIEW

File: tests/test_generator.py [hunk 1]
```diff
@@ -150,16 +150,23 @@ def test_run(self):
         generator_obj = Generator("blog",
             "Article",
             ("title:charfield", "body:textfield"))
-        generator_obj.core_dir = self.core_folder
-        generator_obj.api_dir = self.api_folder
-        result = generator_obj.run()
-        assert result == print("🎉 Your RESTful Article api resource is ready 🎉")
+        result = self._extracted_from_test_run_6(
```
Comment (yezz123) [inline tests/test_generator.py:153]: Unused variable (Relate to build Failed) #6 #5

File: tests/test_generator.py [hunk 2]
```diff
@@ -178,28 +185,26 @@ def test_generate_models(self):
         """
         Tests
         """
-        generator_obj = Generator("blog",
-            "Article",
-            ("title:charfield", "body:textfield"))
-        generator_obj.core_dir = self.core_folder
-        generator_obj.api_dir = self.api_folder
-        generator_obj.generate_models()
+        generator_obj = self._extracted_from_test_generate_models_3()
```
Comment (yezz123) [inline tests/test_generator.py:188]: Unused variable `'generator_obj'` #6 #5

**Review by yezz123 — COMMENTED** [review]
I Guess, Extract duplicate code into the method is not achievable by the Test. #5 #6

File: tests/test_generator.py (hunk 1 above)
Comment (yezz123) [reply tests/test_generator.py]: I Just revert `test_run`  to the old version. [48bf7ca](https://github.com/Abdenasser/dr_scaffold/pull/5/commits/48bf7ca4ea399e4ef7fa7b84c6aa208ae8478652)

**Review by Abdenasser — APPROVED** [review]
@yezz123 LGTM, thanks a lot 👍🏻

**Pull request merged**